website/blog/2018-12-21-state-of-react-native-community.md

```diff
@@ -0,0 +1,51 @@
+---
+title: The State the React Native Community in 2018
+author: Lorenzo Sciandra
+authorTitle: Core Maintainer & React Native Developer
+authorURL: https://github.com/kelset
+authorImageURL: https://avatars2.githubusercontent.com/u/16104054?s=460&v=4
+authorTwitter: kelset
+category: announcements
+---
+
+Two years from now, we’ll probably look back at 2018 as a turning point in the React Native development history.
+
+A lot of eyes are currently pointed at the incoming rewrite of the architecture, widely known as [Fabric](https://github.com/react-native-community/discussions-and-proposals/issues/4), which will enable our mobile apps to reach new peaks of mobile performances (alongside JSI & TurboModules, which are equally as important to reach said goal).
+
+But I will argue that the biggest shift initiated during 2018 is not code related;  this title should go to the empowerment that the React Native Community gained.
+
+Since the very beginning, Facebook allowed developers from all around the world to participate in this Open Source project; investing a few super active members of the community of responsibilities like owning the release cycle.
```

Comment:
"From the beginning Facebook encouraged developers from all around the world to participate in React Native's open source project. Since then, a number of core contributors emerged who handle, among other things, the release process" ?

Reply:
👌